from PyQt5 import QtCore, QtGui, QtWidgets

class Ui_BuyBestHouse(object):
    def setupUi(self, BuyBestHouse):
        BuyBestHouse.setObjectName("BuyBestHouse")
        BuyBestHouse.resize(390, 440)
        BuyBestHouse.setMaximumSize(QtCore.QSize(390, 440))
        self.verticalLayoutWidget = QtWidgets.QWidget(BuyBestHouse)
        self.verticalLayoutWidget.setGeometry(QtCore.QRect(60, 40, 261, 361))
        self.verticalLayoutWidget.setObjectName("verticalLayoutWidget")
        self.formLayout = QtWidgets.QVBoxLayout(self.verticalLayoutWidget)
        self.formLayout.setContentsMargins(0, 0, 0, 0)
        self.formLayout.setSpacing(0)
        self.formLayout.setObjectName("formLayout")
        self.sizeLabel = QtWidgets.QLabel(self.verticalLayoutWidget)
        self.sizeLabel.setMaximumSize(QtCore.QSize(16777215, 20))
        self.sizeLabel.setObjectName("sizeLabel")
        self.formLayout.addWidget(self.sizeLabel)
        self.sizeTextEdit = QtWidgets.QTextEdit(self.verticalLayoutWidget)
        self.sizeTextEdit.setMaximumSize(QtCore.QSize(16777215, 30))
        self.sizeTextEdit.setObjectName("sizeTextEdit")
        self.formLayout.addWidget(self.sizeTextEdit)
        self.priceLabel = QtWidgets.QLabel(self.verticalLayoutWidget)
        self.priceLabel.setMaximumSize(QtCore.QSize(16777215, 20))
        self.priceLabel.setObjectName("priceLabel")
        self.formLayout.addWidget(self.priceLabel)
        self.priceTextEdit = QtWidgets.QTextEdit(self.verticalLayoutWidget)
        self.priceTextEdit.setMaximumSize(QtCore.QSize(16777215, 30))
        self.priceTextEdit.setObjectName("priceTextEdit")
        self.formLayout.addWidget(self.priceTextEdit)
        self.rentPriceLabel = QtWidgets.QLabel(self.verticalLayoutWidget)
        self.rentPriceLabel.setMaximumSize(QtCore.QSize(16777215, 20))
        self.rentPriceLabel.setObjectName("rentPriceLabel")
        self.formLayout.addWidget(self.rentPriceLabel)
        self.rentPriceTextEdit = QtWidgets.QTextEdit(self.verticalLayoutWidget)
        self.rentPriceTextEdit.setMaximumSize(QtCore.QSize(16777215, 30))
        self.rentPriceTextEdit.setObjectName("rentPriceTextEdit")
        self.formLayout.addWidget(self.rentPriceTextEdit)
        self.bedroomCountLabel = QtWidgets.QLabel(self.verticalLayoutWidget)
        self.bedroomCountLabel.setMaximumSize(QtCore.QSize(16777215, 20))
        self.bedroomCountLabel.setObjectName("bedroomCountLabel")
        self.formLayout.addWidget(self.bedroomCountLabel)
        self.bedroomCountTextEdit = QtWidgets.QTextEdit(self.verticalLayoutWidget)
        self.bedroomCountTextEdit.setMaximumSize(QtCore.QSize(16777215, 30))
        self.bedroomCountTextEdit.setObjectName("bedroomCountTextEdit")
        self.formLayout.addWidget(self.bedroomCountTextEdit)
        self.furnishedLabel = QtWidgets.QLabel(self.verticalLayoutWidget)
        self.furnishedLabel.setMaximumSize(QtCore.QSize(16777215, 20))
        self.furnishedLabel.setObjectName("furnishedLabel")
        self.formLayout.addWidget(self.furnishedLabel)
        self.furnishedTextEdit = QtWidgets.QTextEdit(self.verticalLayoutWidget)
        self.furnishedTextEdit.setMaximumSize(QtCore.QSize(16777215, 30))
        self.furnishedTextEdit.setObjectName("furnishedTextEdit")
        self.formLayout.addWidget(self.furnishedTextEdit)
        self.bestHouseTypeLabel = QtWidgets.QLabel(self.verticalLayoutWidget)
        self.bestHouseTypeLabel.setMaximumSize(QtCore.QSize(16777215, 20))
        self.bestHouseTypeLabel.setObjectName("bestHouseTypeLabel")
        self.formLayout.addWidget(self.bestHouseTypeLabel)
        self.bestHouseTypeComboBox = QtWidgets.QComboBox(self.verticalLayoutWidget)
        self.bestHouseTypeComboBox.setMaximumSize(QtCore.QSize(16777215, 30))
        self.bestHouseTypeComboBox.setEditable(False)
        self.bestHouseTypeComboBox.setCurrentText("")
        self.bestHouseTypeComboBox.setMaxVisibleItems(2)
        self.bestHouseTypeComboBox.setMaxCount(2)
        self.bestHouseTypeComboBox.setObjectName("bestHouseTypeComboBox")
        self.formLayout.addWidget(self.bestHouseTypeComboBox)
        self.searchHouseButton = QtWidgets.QPushButton(self.verticalLayoutWidget)
        self.searchHouseButton.setObjectName("searchHouseButton")
        self.formLayout.addWidget(self.searchHouseButton)

        self.retranslateUi(BuyBestHouse)
        QtCore.QMetaObject.connectSlotsByName(BuyBestHouse)

    def retranslateUi(self, BuyBestHouse):
        _translate = QtCore.QCoreApplication.translate
        BuyBestHouse.setWindowTitle(_translate("BuyBestHouse", "Saha Housings - Buy the Bes House"))
        self.sizeLabel.setText(_translate("BuyBestHouse", "Size"))
        self.priceLabel.setText(_translate("BuyBestHouse", "price"))
        self.rentPriceLabel.setText(_translate("BuyBestHouse", "Rent price"))
        self.rentPriceTextEdit.setPlaceholderText(_translate("BuyBestHouse", "(Type 0 if your not looking for renting)"))
        self.bedroomCountLabel.setText(_translate("BuyBestHouse", "Bedroom Count"))
        self.furnishedLabel.setText(_translate("BuyBestHouse", "Furnished"))
        self.furnishedTextEdit.setPlaceholderText(_translate("BuyBestHouse", "(True or False)"))
        self.bestHouseTypeLabel.setText(_translate("BuyBestHouse", "Best house type"))
        self.bestHouseTypeComboBox.setPlaceholderText(_translate("BuyBestHouse", "Your best house by?"))
        self.searchHouseButton.setText(_translate("BuyBestHouse", "Add House"))